### PR TITLE
Fix #269952: show new templates in new score wizard w/o restart

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
replaces #3773 with a much simpler approach (thanks @MarcSabatella)

### CODE REVIEW

File: mscore/file.cpp [hunk 1]
```diff
@@ -493,8 +493,7 @@ void MuseScore::updateNewWizard()
 
 void MuseScore::newFile()
       {
-      if (newWizard == 0)
-            newWizard = new NewWizard(this);
+      newWizard->updateNewWizard();
```
Comment (MarcSabatella) [inline mscore/file.cpp]: not "else"?

Comment (Jojo-Schmitz) [reply mscore/file.cpp]: No else needed, it is all done inside `updateNewWizard()` already

Comment (Jojo-Schmitz) [reply mscore/file.cpp]: the `newWizard->` was wrong though

**Comment by Jojo-Schmitz** [comment]
Travis will fail, due to being in RELEASE mode

File: mscore/file.cpp [hunk 2]
```diff
@@ -493,8 +493,7 @@ void MuseScore::updateNewWizard()
 
 void MuseScore::newFile()
       {
-      if (newWizard == 0)
-            newWizard = new NewWizard(this);
+      updateNewWizard();
```
Comment (MarcSabatella) [inline mscore/file.cpp]: My reason for suggesting the "else" is that this shouldn't be needed in the case where we are creating the wizard for the first time - if it were, then things never would have worked.  Can't build right now to test for myself, but I would think we're now building template list twice the first time wizard is invoked.

Comment (Jojo-Schmitz) [reply mscore/file.cpp]: newFile() instatiated newwizzard, if need be, updateNewWizzard() does that too, and now only it.

Comment (MarcSabatella) [reply mscore/file.cpp]: Never mind, I see now that the previous two lines were removed; I missed that on first read.  That's why I thought something was happening twice.

**Comment by Jojo-Schmitz** [comment]
Can we cherry-pick this for master?

**Pull request merged**